js/index.js

```diff
@@ -0,0 +1,19 @@
+var today = new Date();
+var thisYear = today.getFullYear();
+var footer = document.querySelector('footer');
+var copyright = document.createElement('p');
```

Comment:
The `var` keyword in JavaScript is used to declare a variable. However, it has some characteristics that can lead to unexpected behavior and make code harder to maintain. 
For example if you use or modify the `today` variable (line 1) in other function, you're going to modify that variable, instead you should use let keyword that has block-level scope. and if `let today` is created in other part of the code (inside another function) both will be independent.

Also remember to use `const` if the variable value isn't gonna change. (Like you did in then following lines)